code/modules/wiremod/components/ntnet/ntnet_request.dm

```diff
@@ -0,0 +1,39 @@
+/**
+ * # NTNet Transmitter Component
+ *
+ * Sends a data package through NTNet
+ */
+
+/obj/item/circuit_component/ntnet_send
+	display_name = "NTNet Transmitter"
+	desc = "Sends a data package through NTNet when triggered. If target HID is not provided, data will be sent to all circuits in the network. If Encryption Key is set then transmitted data will be only picked up by receivers with the same Encryption Key."
+
+	circuit_flags = CIRCUIT_FLAG_INPUT_SIGNAL
+
+	network_id = __NETWORK_CIRCUITS
+
+	var/datum/port/input/target_hid
+	var/datum/port/input/data_package
+	var/datum/port/input/secondary_package
+	var/datum/port/input/enc_key
+
+/obj/item/circuit_component/ntnet_send/Initialize()
+	. = ..()
+	target_hid = add_input_port("Target Hardware ID", PORT_TYPE_STRING)
```

Comment:
Same here, it should be treating NTNet as a NETWORK, not as a way to directly communicate between TWO circuits.